assets/proc.js

```diff
@@ -439,26 +441,28 @@ function doPopupFor(img) {
     $('#image_info_modal').modal('toggle');
 }
 
-function updateTitleStickyDirect() {
-    const height = Math.round(document.getElementById('top_nav_bar').getBoundingClientRect().height);
-    const header = document.getElementById('image_table_header');
-    if (header.style.top !== height + 'px') { // This check is to reduce the odds of the browser yelling at us
+function updateTitleStickyDirect(topBar) {
+    // client rect is dynamically animated, so, uh, just hack it for now.
+    // could listen to `transitionend` or bootstrap events, but would create an artifact
+    setTimeout(() => {
+        const height = Math.round(topBar.getBoundingClientRect().height);
+        const header = document.getElementById('image_table_header');
+        if (header.style.top === (height + 'px')) {
+            return;
+        }
         header.style.top = height + 'px';
-    }
+        updateTitleStickyDirect(topBar);
```

Comment:
I believe this code change might reintroduce already-fixed bugs. Browsers are really upsettable about rapid changes in CSS (eg Firefox will just throw a console warning and disable sticky entirely if you spam too much - this could be triggered in earlier versions of this code by just spamclicking the accordion button)